src/lib/storage/dictionary_column.cpp

```diff
@@ -67,10 +72,11 @@ std::shared_ptr<const BaseAttributeVector> DictionaryColumn<T>::attribute_vector
 
 // TODO(anyone): This method is part of an algorithm that hasn’t yet been updated to support null values.
 template <typename T>
-const pmr_concurrent_vector<T> DictionaryColumn<T>::materialize_values() const {
-  pmr_concurrent_vector<T> values(_attribute_vector->size(), T(), _dictionary->get_allocator());
+const pmr_concurrent_vector<std::optional<T>> DictionaryColumn<T>::materialize_values() const {
+  pmr_concurrent_vector<std::optional<T>> values(_attribute_vector->size(), std::nullopt, _dictionary->get_allocator());
 
   for (ChunkOffset chunk_offset = 0; chunk_offset < _attribute_vector->size(); ++chunk_offset) {
+    if (is_null(chunk_offset)) continue;
```

Comment:
So the TODO comment above can be removed?

Reply:
Yes. Done.